resources/views/searches/resources/groups/edit_group.blade.php

```diff
@@ -0,0 +1,94 @@
+                                    {{ __('group.vehicle') }}:
+                                </div>
+                                {{ Form::text('edit_vehicle', null , array('class' => 'form-control')) }}
+                            </div>
+                        </div>
+                        <div class="row margin-top-sm">
+                            <div class="col-6">
+                                <div class="text-left">
+                                    {{ __('group.broadcast') }}:
+                                </div>
+                                {{ Form::text('edit_broadcast', null, array('class' => 'form-control')) }}
+                            </div>
+                            <div class="col-6">
+                                <div class="text-left">
+                                    {{ __('group.gps') }}:
+                                </div>
+                                {{ Form::text('edit_gps', null, array('class' => 'form-control')) }}
+                            </div>
+                        </div>                
+                        <div class="row margin-top-sm">
+                            <div class="col-12">
+                                <div class="text-left">
+                                    {{ __('group.people_involved') }}:
+                                </div>
+                                {{ Form::textarea('edit_people_involved', null, array('class' => 'form-control','rows'=> 3, 'resize' => 'none')) }}
+                            </div>                        
+                        </div>  
+                    </div>   <!-- col-12 - CLOSE -->              
+                </div> <!-- Modal body - CLOSE -->
+                <div id="edit_error_message_container" class="alert alert-danger alert-dismissible fade show" role="alert" style="display:none">
+                    <div class="container text-center">
+                        <a href="#" class="close" data-dismiss="alert" aria-label="close">&times;</a>
+                        <p id="edit_error_message">Error goes here !</p>
+                    </div>
+                </div>
+                <!-- Modal footer - OPEN -->
+                <div class="modal-footer">
+                    <a class="btn btn-light" data-dismiss="modal">
+                        {{ __('actions.cancel') }}
+                    </a>
+                    <!-- Register button - OPEN -->
+                    <button type="button" class="btn btn-primary" id="btn_edit_group">
+                        {{ __('actions.edit') }}
+                    </button>
+                    <!-- Register button - CLOSE -->
+                </div>
+                <!-- Modal footer - CLOSE -->            
+            </form>
+            <!-- Form register - CLOSE -->
+        </div>
+        <!-- Modal content - CLOSE -->     
+    </div>
+    <!-- Modal dialog - CLOSE -->   
+</div>
+<!-- Add group modal - CLOSE -->
```

Comment:
Should be <!-- Edit group modal - CLOSE -->

Reply:
ok